src/GraphQL/Types/DateGraphType.cs

```diff
@@ -17,13 +18,27 @@ public override object Serialize(object value)
 
             if (date is DateTime dateTime)
             {
-                return dateTime.ToString("yyyy-MM-dd");
+                return dateTime.ToString("yyyy-MM-dd", CultureInfo.InvariantCulture);
             }
 
             return null;
         }
 
-        public override object ParseValue(object value) => ValueConverter.ConvertTo(value, typeof(DateTime));
+        public override object ParseValue(object value)
+        {
+            if (value is DateTime dateTime)
+            {
+                return dateTime;
```

Comment:
Wouldn’t it be right to take only the part responsible for the date? Or it’s generally worth generating an error if the format of the input data does not match the required format for `Date`. I believe it is wrong to allow the use of a date with a time component for a date scalar

Reply:
fixed to throw an exception